packages/embed/src/overlay/overlay.css

```diff
@@ -49,3 +49,41 @@
   display: block;
   cursor: pointer;
 }
+
+.gr4vy__lightbox {
+  background: white;
+  padding: 16px;
+}
+
```

Comment:
The following represent all the sizes available on a challenge.
Nothing fancy here - we just check there is room for 100px padding on width/height from the page.

These are:
- 250x400
- 390x400
- 500x600
- 600x400